src/actions/profileActions.js

```diff
@@ -0,0 +1,74 @@
+import uploadImageCloudinary from '../utilities/cloudinaryUpload';
+import { Put, Get } from '../utilities/apiRequests';
+import {
+  GET_A_USER,
+  TOGGLE_BUTTON,
+  UPDATE_FIELD,
+  CLEAR_EDIT
+} from '../actionTypes';
+
+const fetchAUser = user => ({
+  type: GET_A_USER,
+  user
+});
+
+export const editFields = (name, value) => ({
+  type: UPDATE_FIELD,
+  name,
+  value
+});
+
+export const toggleButton = () => ({
+  type: TOGGLE_BUTTON
+});
+
+export const clearEdit = status => ({
+  type: CLEAR_EDIT,
+  status
+});
+
+const getFollowDetails = async (userId) => {
+  try {
+    const followDetails = await Get(`/users/${userId}/follow`);
+    const followingCount = followDetails.followingDetails.count;
```

Comment:
We don't need these variables IMO. You can reference them directly in the return statement.